### PR TITLE
CI: Added audio tests

### PR DESCRIPTION
**Pull request by koffes**
Added Audio in CI when SDC or LC3 changes

### CODE REVIEW

File: .github/test-spec.yml [hunk 1]
```diff
@@ -180,3 +180,9 @@
       - "crypto/**/*"
       - "!crypto/doc/**/*"
       - "!crypto/*.rst"
+
+"CI-audio-test":
+  - "lc3/**/*"
+  - "softdevice_controller/CMakeLists.txt"
```
Comment (Tschet1) [inline .github/test-spec.yml:186]: nitpick: not sure if this line is needed

**Review by Tschet1 — APPROVED** [review]

**Review by alexsven — APPROVED** [review]

File: .github/test-spec.yml [hunk 2]
```diff
@@ -180,3 +180,9 @@
       - "crypto/**/*"
       - "!crypto/doc/**/*"
       - "!crypto/*.rst"
+
+"CI-audio-test":
```
Comment (koffes) [inline .github/test-spec.yml:184]: @Frodevan : Is there a way CI can run with this controller when changes are pushed to this repo?

**Comment by alexsven** [comment]
Tested in https://github.com/nrfconnect/sdk-nrf/pull/13069

File: .github/test-spec.yml (hunk 2 above)
Comment (alexsven) [reply .github/test-spec.yml:184]: This has now been fixed for CI

**Pull request merged**